docs/reference/execute-methods.md

```diff
@@ -167,6 +167,7 @@ Name | Type | Required | Description | Example
 app | string | yes | See the description of the `appium:app` capability | /path/to/my.app
 timeoutMs | number | no | The maximum time to wait until app install is finished in milliseconds on real devices. If not provided then the value of `appium:appPushTimeout` capability is used. If the capability is not provided then equals to 240000ms | 500000
 strategy | string | no | One of possible app installation strategies on real devices. This argument is ignored on simulators. If not provided then the value of `appium:appInstallStrategy` is used. If the latter is also not provided then `serial` is used. See the description of `appium:appInstallStrategy` capability for more details on available values. | parallel
+checkVersion | bool | no | If set to `true`, it will make xcuitest driver to verify whether the app version currently installed on the device under test is older than the one, which is provided as `app` value. No app reinstall is going to happen if the candidate app has the same or older version number than the already installed copy of it. The version number used for comparison must be provided as [CFBundleVersion](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion) [Semantic Versioning](https://semver.org/)-compatible value in the application's `Info.plist`. No validation is performed and the `app` is installed if `checkVersion` was not provided or `false`, which is default behavior. | true
```

Comment:
I assume if one would try to install an older (or same) version of an existing app without uninstalling it first then an installation error will be thrown by the system.
Same with Android

Reply:
Maybe possibly only the older ones could return an error. In a new session request, I saw an older one returned an error in a real device. The same one succeeded.

Reply:
For real devices, the CFBundleVersion/short version did not affect the installation. So any errors never occurred while installing to lower CFBundleVersion/newer CFBundleVersion.